### PR TITLE
MMI-4841-clean-response-dont-return-empty-responses

### PR DESCRIPTION
**Pull request by jpsains**

### CODE REVIEW

File: src/adapters/morpho-aave-v3-eth/tests/snapshots/ethereum.profits.3.json
Comment (bergarces) [inline src/adapters/morpho-aave-v3-eth/tests/snapshots/ethereum.profits.3.json:1]: We should probably remove this test. Or add addresses that produce something.

File: src/adapters/gmx/tests/snapshots/arbitrum.deposits.json
Comment (bergarces) [inline src/adapters/gmx/tests/snapshots/arbitrum.deposits.json:1]: Is this intended?

**Review by bergarces — APPROVED** [review]

File: src/adapters/gmx/tests/snapshots/arbitrum.deposits.json
Comment (jpsains) [reply src/adapters/gmx/tests/snapshots/arbitrum.deposits.json:1]: yeah, it looks like an empty profits response to me?

Comment (jpsains) [reply src/adapters/gmx/tests/snapshots/arbitrum.deposits.json:1]: ah the price missing ha...no

File: src/adapters/morpho-aave-v3-eth/tests/snapshots/ethereum.profits.3.json
Comment (jpsains) [reply src/adapters/morpho-aave-v3-eth/tests/snapshots/ethereum.profits.3.json:1]: Ill create ticket to investigate

**Pull request merged**